SingleViewApi/Startup.cs

```diff
@@ -359,10 +365,10 @@ public static void Configure(IApplicationBuilder app, IWebHostEnvironment env, I
                 // SwaggerGen won't find controllers that are routed via this technique.
                 endpoints.MapControllerRoute("default", "{controller=Home}/{action=Index}/{id?}");
 
-                endpoints.MapHealthChecks("/api/v1/healthcheck/ping", new HealthCheckOptions()
-                {
-                    ResponseWriter = HealthCheckResponseWriter.WriteResponse
-                });
+                // endpoints.MapHealthChecks("/api/v1/healthcheck/ping", new HealthCheckOptions()
+                // {
+                //     ResponseWriter = HealthCheckResponseWriter.WriteResponse
+                // });
```

Comment:
restore this

Reply:
@elena-vi does this need to be restored?